pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -162,6 +162,13 @@ def _export(
     bytesio = io.BytesIO()
     opset_ver = kwargs.get('opset_version', None)
     force_verbose = False
+
+    if pytorch_pfn_extras.requires("1.13.0"):
+        if "training" in kwargs and (isinstance(kwargs["training"], bool) or kwargs['training'] is None):
+            kwargs["training"] = torch.onnx.TrainingMode.TRAINING \
+                if kwargs["training"] \
+                else torch.onnx.TrainingMode.TRAINING
```

Comment:
I think this is setting it to the same value always, right?

Reply:
Thank you! This seems to be bug. Will fix it!